lib/rules/web/ocm_console/cluster_list_page.xyaml

```diff
@@ -341,6 +358,212 @@ go_to_disconnected_cluster_register_page:
+        - selector:
+            xpath: //div[@class='metrics-chart']
+check_osd_details_in_detail_page:
+    elements:
+        - selector:
+            xpath: //h2[@class='pf-c-title pf-m-3xl' and text()='Details']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Cluster ID']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Type']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dd[text()='OSD']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Location']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Provider']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dd[text()='AWS']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Version']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Operating system']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Created at']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Owner']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Billing Model']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dd[text()='Standard']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Status']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dd[text()='ready']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Total vCPU']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Total Memory']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Load Balancers']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Persistent Storage']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Desired Nodes']
+        - selector:
+            xpath: //dl[@class='cluster-details-item-list left']/dt[text()='Master:']
+        - selector:
+            xpath: //dl[@class='cluster-details-item-list left']/dt[text()='Infra:']
+        - selector:
+            xpath: //dl[@class='cluster-details-item-list left']/dt[text()='Compute:']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Actual Nodes']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Network']
+expand_actions_behind_open_console_button:
+    element:
+        selector:
+            xpath: //span[@class='pf-c-dropdown__toggle-text' and text()='Actions']/../../button
+        op: click
+check_ocp_actions_behind_open_console_button:
```

Comment:
All of these actions should not be defined in cluster_list_page.xyaml. They should be defined in cluster_detail.xyaml

Reply:
Updated